icicle/appUtils/ntt/kernel_ntt.cu

```diff
@@ -1025,9 +1025,27 @@ namespace ntt {
     bool columns_batch,
     bool is_inverse,
     bool fast_tw,
-    Ordering ordering,
+    ntt::Ordering ordering,
     curve_config::scalar_t* arbitrary_coset,
     int coset_gen_index,
     cudaStream_t cuda_stream);
 
-} // namespace ntt
+#if defined(ECNTT_DEFINED)
```

Comment:
Comment this whole ifdef for now

Reply:
ok, btw - it compiles fast in .cu without Rust or Go

Reply:
We agreed for the time being to remove mixed_radix from ecntt